documentation/modules/rn-2.2.adoc

```diff
@@ -40,6 +52,15 @@ As a temporary workaround, the `Importer` pod is removed at the end of the preco
 
 This issue only affects the `Importer` pod log and warm migration. Cold migration and the `virt-v2v` logs are not affected.
 
+.Unclear error message for failed migration of VMs with no operation system
+
+When migration plans for VMs with no operation system fail, the error message does not indicate the reason for the failure. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2008846[*BZ#2008846*])
+
+.In a migration plan created from the CLI using names instead of IDs for network and storage mapping and for VMs, the mapping is shown as unavailable when the plan is opened in the {project-full} web console.
```

Comment:
Way too verbose. Fluff. Change to this:

`Network, storage, and VM _names_ in the "Plan" CR are not displayed in the web console`

Reply:
But the explanation should mention that the CLI does recognize names, whereas the console does not.